### PR TITLE
Migrating comments and comment list to a generic space for use in mug…

### PR DESCRIPTION
**Pull request by eilinora**
… comm and event testing

### CODE REVIEW

**Review by sadafie — APPROVED** [review]
LGTM, pending build issue

File: src/group/communication/app.js [hunk 1]
```diff
@@ -1,9 +1,11 @@
 import {
 	MOCK_GROUP_COMMUNICATION,
-	MOCK_GROUP_COMMUNICATION_COMMENTS,
 	MOCK_GROUP_COMMUNICATION_FOLLOWERS,
 	MOCK_GROUP_COMMUNICATION_SUGGESTED_FOLLOWERS
 } from './api';
+import {
+	MOCK_COMMENTS_LIST,
+} from './../api';
```
Comment (mmcgahan) [inline src/group/communication/app.js:8]: this is missing a dot!!! will fix and merge to master

Comment (eilinora) [inline src/group/communication/app.js]: mmm no :) it's correct. it's up one folder from itself... maybe we could delete `./` instead

Comment (eilinora) [reply src/group/communication/app.js:8]: don't listen to me :)

Comment (mmcgahan) [reply src/group/communication/app.js:8]: there is nothing in `src/group/`: https://github.com/meetup/meetup-web-mocks/tree/master/src/group

Comment (eilinora) [reply src/group/communication/app.js:8]: it's weird the build didn't fail :/

Comment (mmcgahan) [reply src/group/communication/app.js:8]: The build for this repo is just a Babel transpile - no code is actually executed.

I guess we probably should add some kind of basic unit test, but import errors are currently silent.

**Pull request merged**